node.go

```diff
@@ -342,7 +342,27 @@ func (n *Node) LoadCommandClassVersions() error {
 			return err
 		}
 
-		time.Sleep(interviewTimeout)
+		// Record time that we started waiting for a version report
+		waitingForVersionSince := time.Now()
+
+		// Wait until either the version has been reported, or we've waited too long
+		// 5 seconds is used to be sure any collisions (and thus CAN timeouts) are over
+		for {
+			// Check to see if this command class has been reported
+			if v := n.CommandClasses.GetVersion(commandClassID); v != 0 {
+				break
+			}
+
+			if waitingForVersionSince.Add(time.Second * 5).Before(time.Now()) {
+				n.client.l.Warn("stopped waiting for confirmation of version command class",
+					zap.String("command class ID", fmt.Sprintf("%02x", commandClassID)),
+				)
+				break
+			}
+
+			time.Sleep(versionReportedQueryPeriod)
```

Comment:
I don't love that we're busy waiting here, but if this is what's required without more of a refactor then it's cool.